packages/@lwc/ssr-compiler/src/compile-template/index.ts

```diff
@@ -55,7 +56,9 @@ const bExportTemplate = esTemplate`
         if (!isLightDom) {
             yield '</template>';
             if (shadowSlottedContent) {
-                yield* shadowSlottedContent();
+                // instance must be passed in; this is used to establish the contextful relationship
+                // between context provider (aka parent component) and context consumer (aka slotted content)
+                yield* shadowSlottedContent(instance);
```

Comment:
It's really weird that we pass in the `instance` here but the `contextualParent` when calling a light DOM slotted content generator... Something to investigate later. (Maybe we can break this with several layers of context providers / components.)

Reply:
This is in the root `tmpl` function, so at this point in the code, `contextfulParent` will _always_ be `instance`. It doesn't matter which we use, but we can use `contextfulParent` for consistency.
```suggestion
                yield* shadowSlottedContent(contextfulParent);
```

Reply:
Whew okay, that makes sense!